octavia-cli/README.md

```diff
@@ -26,7 +26,8 @@ This script:
 
 ## 2.b If you want to directly run the CLI without alias in your current directory:
 ```bash
-docker run --rm -v ${PWD}:/home/octavia-project --network host -e AIRBYTE_URL="${AIRBYTE_URL}" -e AIRBYTE_WORKSPACE_ID="${AIRBYTE_WORKSPACE_ID}" airbyte/octavia-cli:dev
+mkdir my_octavia_project_directory # Create your octavia project directory where YAML configurations will be stored.
```

Comment:
```suggestion
mkdir -p my_octavia_project_directory # Create your octavia project directory where YAML configurations will be stored.
```
I think the flag `-p` is useful 📦

Reply:
There's no subdir in this example so no need to create directories recursively IMO, wdyt? 🤔